src/blog/featured.ts

```
@@ -9,7 +9,18 @@ const FEATURED_TITLES = [
 ];
 
 export function getFeaturedWritings(allWritings: WritingItem[]): WritingItem[] {
-  return FEATURED_TITLES.map((title) =>
-    allWritings.find((w) => w.title.startsWith(title)),
-  ).filter((w): w is WritingItem => w != null);
+  const featured: WritingItem[] = []
```

ESLint

[error] Insert `;`

(prettier/prettier)